### DISCUSSION TITLE
Is it possible to use the bindings in /server routes?

### DISCUSSION THREAD

**Discussion by luishugoooo** [vuejs/vuefire#1542] [Questions and Help]
.

**Comment by sugoidesune** [comment]
I have no clue what you need. Your very elaborative "." didn't help.

**Reply by luishugoooo** [reply]
I'm sorry, I was a bit in a rush and did not think about the question properly. I just want to use useFirestore(), useCollection() etc. in [Nuxt /server routes ](https://nuxt.com/docs/guide/directory-structure/server). Is that possible?
